scripts/build-rpm.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+source /etc/os-release
+
+distro="$ID"
+project_dir="$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)"
+version="$(git describe --tags | cut -f 1 -d "-" | sed 's/^v//')"
+release="dev_$(git describe --tags --dirty --always | sed 's/^v//g' | sed 's/-/_/g')"
+output_dir="${project_dir}/out"
+
+echo "project root: $project_dir"
+echo "build version: $version"
+echo "build release: $release"
+
+set -x
+
+# Create rpmbuild directory layout.
+build_dir="$(mktemp -d)"
+export HOME="$build_dir"
+mkdir -p "${build_dir}"/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+
+# Azure Linux / Mariner requires rpmmacros.
+if [[ $ID == "mariner" || $ID == "azurelinux" ]]; then
```

Comment:
Similar to the version and release macros I suggested above, you should be able to use `--define "_topdir ..."` when you build the RPM.

Reply:
great, thanks! done